holmes/core/investigation_structured_output.py

```diff
@@ -1,6 +1,16 @@
-from typing import Any, Dict
+from typing import Any, Dict, Optional, Tuple, Union
+import json
 
-DEFAULT_SECTIONS = {
+from pydantic import  RootModel
+
+InputSectionsDataType = Dict[str, str]
+
+OutputSectionsDataType = Optional[Dict[str, Union[str, None]]]
+
+class SectionsData(RootModel):
```

Comment:
Consider to use RootModel instead of having model with root, https://docs.pydantic.dev/2.10/concepts/models/#rootmodel-and-custom-root-types

WDYT?

Reply:
:heavy_check_mark: